src/utils/date.ts

```diff
@@ -64,13 +78,26 @@ export function secondsToMinutesLong(
   });
 }
 
+type HumanizerLanguage = 'no' | 'nn' | 'en';
+
+const languageToHumanizerLanguage = (language: Language): HumanizerLanguage => {
+  switch (language) {
+    case Language.Norwegian:
+      return 'no';
+    case Language.English:
+      return 'en';
+    case Language.Nynorsk:
+      return 'nn';
+  }
+};
```

Comment:
Is this strictly necessary? Looks like as long as our `Language` enum resolves to a sensible two-letter language code ([ISO 639-1 to be exact!](https://github.com/EvanHahn/HumanizeDuration.js/tree/main#language)), we could pass the raw language value to the humanizer below. 

Or is there any downsides to that?

Reply:
I tried passing the language prop directly in `secondsToDuration` and `getHumanizer`, it ended with `No language found` error. 

I think it might be because language in our case might be defined as `nb`, while [the library](https://github.com/EvanHahn/HumanizeDuration.js/blob/dcd80f80a380f3bd3bb026927a7a414ceefc76f3/humanize-duration.js#L902) and ISO 639-1 seems to expect `no` for bokmål. Potentially an obstacle for having a PR to the library 😄 

![IMG_5739](https://github.com/AtB-AS/mittatb-app/assets/21310942/20585007-1904-4214-9d3d-41aa198c6c29)

Reply:
Aha, "macrolanguages" are of course a thing 😆 

![Screenshot 2023-08-09 at 15 12 16](https://github.com/AtB-AS/mittatb-app/assets/1774972/0e80d83c-bbab-45f3-a93a-857d910700c3)

Reply:
Of course, this is why we can't have nice things! 😄